plugins/plugin-typescript/plugin.js

```diff
@@ -22,11 +22,18 @@ function typescriptPlugin(snowpackConfig, {tsc, args} = {}) {
           log('WORKER_RESET', {});
           stdOutput = stdOutput.replace(/\x1Bc/, '').replace(/\u001bc/, '');
         }
-        log('WORKER_MSG', {level: 'log', msg: stdOutput});
+        log('WORKER_MSG', {msg: stdOutput});
       }
       stdout && stdout.on('data', dataListener);
       stderr && stderr.on('data', dataListener);
-      return workerPromise;
+      return workerPromise.catch((err) => {
```

Comment:
This is throwing in CI, looks like this is not actually a Promise? Odd.

Reply:
Fixed!